artifacts/kernel.json

```diff
@@ -0,0 +1,8 @@
+{
+  "argv": [
+    "kubeflow-kernel",
+    "{connection_file}"
+  ],
+  "display_name": "Kubeflow",
```

Comment:
s/Kubeflow/Kubeflow-PythonN?

Reply:
I think we will support Python 2 & 3 in the future, while now we can set to Kubeflow-Python3